model-optimizer/extensions/front/onnx/non_max_suppression_normalize.py

```diff
@@ -30,10 +30,10 @@ class NonMaxSuppressionNormalize(FrontReplacementSubgraph):
 
     def find_and_replace_pattern(self, graph: Graph):
         for nms in graph.get_op_nodes(op='NonMaxSuppression'):
-            # make inputs 2 to 4 to have shape [] instead of [1] (convert 1D to 0D)
+            # make inputs 2 to 4 to have shape [1] instead of [0] (convert 0D to 1D)
+            nms_name = nms.soft_get('name', nms.id)
             for port_id in range(2, 5):
```

Comment:
5 must be changed to 6 since inputs with indices 2, 3, 4 and 5 must be updated

Reply:
Fixed.